recipes/robustness/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "robustness" %}
+{% set version = "1.2.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 82b1cae2a5f16846fd4caadab9cd43fda14f752feec76cc79c38be185255ce99
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not linux]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cox
+    - dill
+    - gitpython
+    - gputil
+    - grpcio
+    - matplotlib
```

Comment:
Do you need `matplotlib` or `matplotlib-base` ? If the package doesn't depend on interactive images, but only renders images to files, then you can only use `matplotlib-base`. If you need the full Qt backend, then you should use `matplotlib`.